src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -156,7 +157,7 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
   };
   module.preRun = [];
   module.postRun = [];
-  module.preloadPlugins = [];
+  (module as any).preloadPlugins = [];
```

Comment:
This is an anomaly so I'm not bothered about declaring a type for it. `preloadPlugins` isn't referenced from anywhere else.